parm/gfs/oznHoriz.yaml

```diff
@@ -0,0 +1,216 @@
+datasets:
+  - name: horiz
+    type: MonDataSpace
+    satellite: {{SAT}}
+    sensor: {{SENSOR}}
+    run: {{RUN}}
+
+    type: MonDataSpace
+    control_file:
+      - {{DATA}}/ozn_data/horiz/ompsnp_npp.ges.ctl
+    filenames:
+      - {{DATA}}/ozn_data/horiz/{{SENSOR}}_{{SAT}}.{{RUN}}.{{ PDATE | to_YMDH }}.ieee_d
+      - {{DATA}}/ozn_data/horiz/ompsnp_npp.ges.{{ PDATEm6 | to_YMDH }}.ieee_d
```

Comment:
Guessing these are an oversight since it is in draft?

Reply:
Indeed.  I proved the concept and figured I'd clean up later and forgot about it.   I'll correct that.